Tests/TestDoubles/FileManagerSpy.swift

```diff
@@ -15,7 +15,7 @@ class FileManagerSpy: Spy, FileSystemManager {
     var currentDirectoryPathToReturn: String!
 
     var currentDirectoryPath: String {
-        return currentDirectoryPathToReturn
+        return currentDirectoryPathToReturn ?? "/"
```

Comment:
we should delete this nil coalesce - This test double was intentionally written so that it halts if this value hasn't been set